miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/js/MiniAppSecureStorageDispatcher.kt

```diff
@@ -43,7 +43,7 @@ internal class MiniAppSecureStorageDispatcher(
         this.miniAppSecureStorage = MiniAppSecureStorage(
             context,
             databaseVersion,
-            maxStorageSizeLimit
+            (maxStorageSizeLimit * CONVERT_TO_KB)
```

Comment:
Any reason why we are converting to KB here?

Reply:
@rleojoseph yes, in previous implementation this initialization has not been used, it uses this function [link](https://github.com/rakutentech/android-miniapp/blob/1429479cdefbfc04997c461018c40214202f14ce/miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/js/MiniAppSecureStorageDispatcher.kt#L50-L52) so we didnt face any issue previously. But now we use the initial intialization so if we dont convert it in kb, then db will assume the value 5 MB as 5 kb which is very less and memory will be full very quickly.